src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -142,9 +145,10 @@ public static IEnumerable<SyntaxKind> GetPreprocessorKeywordKinds()
             yield return SyntaxKind.TrueKeyword;
             yield return SyntaxKind.FalseKeyword;
             yield return SyntaxKind.DefaultKeyword;
-            yield return SyntaxKind.HiddenKeyword;
+
             for (int i = (int)SyntaxKind.ElifKeyword; i <= (int)SyntaxKind.RestoreKeyword; i++)
```

Comment:
The `SyntaxKind.ElifKeyword` - `SyntaxKind.RestoreKeyword` already contains `SyntaxKind.HiddenKeyword`, so I removed the duplicate. Found while working on tests